packages/ember-auto-import/ts/webpack.ts

```diff
@@ -89,7 +95,7 @@ window._eai_d = define;
 export default class WebpackBundler extends Plugin implements Bundler {
   private state:
     | {
-        webpack: Compiler;
+        webpack: MultiCompiler;
```

Comment:
I think MultiCompiler means we're running the whole webpack build separately multiple times? That should not be necessary. We can keep adding more entrypoints to the same build instead.